concepticondata/conceptlists.tsv

```diff
@@ -185,3 +185,7 @@ Dunn-2011-146	Dunn, Michael and Burenhult, Niclas and Kruspe, Nicole and Tufvess
 Miedema-1985-40	Miedema, J. and Welling, F. I.	1985		40	basic,areal	English	New Guinea		Miedema1985	Miedema1985	This is a list of 40 items based on the [40-item list by Voorhoeve (1975)](:ref:Voorhoeve-1975-40).	29-52	
 Voorhoeve-1975-40	Voorhoeve, C. L.	1975		40	basic,areal	English	New Guinea		Voorhoeve1975	Voorhoeve1975	A 40 word item list possibly derived from unpublished wordlists of J.C. Anceaux	94-129	
 Dye-1968-120	Dye, W. and Townsend, P. and Townsend, W.	1968		120	basic,areal	English	New Guinea		Dye1968	Dye1968	A 120 item list for a survey of the Sepik languages.	151-152	
+Bromley-1967-110	Bromley, M.	1967		110	basic,areal	English	New Guinea		Bromley1967	Bromley1967	A 110 item list possibly derived from Swadesh for a survey of Grand Valley Dani languages	305-307	
```

Comment:
Sorry, but I just realized: it's a 111 item list. And it really seems it comes from Swadesh, but rather his 100 item list plus the 200 item list, as "breast" s new in the 100 item list. Once we make a cladistic analysis of Swadesh-lists in concepticon (with shared errors, and the like ;-), we will know who copied from whom...